tools/project_manager/show_good_word_suggestions_detail.php

```diff
@@ -119,7 +118,7 @@
     echo "<p><b>" . _("Date") . "</b>: " . strftime($datetime_format,$time) . "<br>";
     echo "<b>" . _("Round") . "</b>: $round &nbsp; | &nbsp; ";
     echo "<b>" . _("Proofreader") . "</b>: " . private_message_link($proofer) . "<br>";
-    echo "<b>" . _("Page") . "</b>: <a href='?projectid=$projectid&amp;imagefile=$page&amp;word=$encWord&amp;timeCutoff=$timeCutoff'>$page</a><br>";
+    echo "<b>" . _("Page") . "</b>: <input type='button' class='page-select' value='$page'><br>";
```

Comment:
Does the `.click()` handler not get run on links? I feel like we should be able to keep this a link and avoid all of the fallout of making this a button.

Reply:
No doubt we can make it a link somehow.